TrackingAntennaV3/main/Accelerometer.hpp

```diff
@@ -0,0 +1,23 @@
+/**
+ * @file   Magnetometer.hpp
+ * @Author Michael Lenover (WARG)
+ * @date   January 25, 2017
+ * @brief  
+ * @see https://www.arduino.cc/en/Reference/Ethernet
+ */
+
+#ifndef ACCELEROMETER
+#define ACCELEROMETER
+
+#define COMPASS_ID 12345
+
+//Accelerometer data struct
+typedef struct {
+  float x;
+  float y;
+  float z;
+} Accelerometer;
```

Comment:
I'd rename this to AcceloremeterData to clarify that the structs represent the data output, and not the actual accelorometer